tests/test_categories.py

```diff
@@ -110,3 +114,29 @@ def test_healthcare_specialities():
 
     apply_healthcare_specialities([HealthcareSpecialities.OCCUPATIONAL], item)
     assert item["extras"] == {"healthcare:speciality": "occupational"}
+
+
+@pytest.fixture
+def payment_methods_enum():
+    return PaymentMethods
+
+
+@pytest.fixture
+def item():
+    return Feature()
+
+
+@pytest.mark.parametrize(
```

Comment:
Can you switch this to using a standard test function instead of pytest wrappers? I'd like to keep the tests as easy to add as possible, and introducing new systems for testing feels like we're adding more complexity than necessary.

Reply:
Makes sense, thank you!